### ISSUE TITLE
Release v1.32.12

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#1705]
:robot: A dedicated issue to run tests and deploy release [v1.32.12](https://github.com/deckhouse/deckhouse/milestone/57).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | gcp | azure | openstack | yandex-cloud | vsphere | static`
  - `git_ref` is a tag or branch: `vX.Y.Z | test-X.Y.Z* | main | release-X.Y`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `containerd | docker`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22 | 1.23`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag or branch: `vX.Y.Z | test-vX.Y.Z* | main | release-X.Y`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [7e8671](https://github.com/deckhouse/deckhouse/commit/7e867192941d0ea15e69a3853707689119ae7f52) in branch **main**:
```
[docs] Change the documentation default page to the Overview page. (#1704)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [7c4450](https://github.com/deckhouse/deckhouse/commit/7c445055fd34b7f49c0341ecd36e2034034d8e00) in branch **main**:
```
[docs] Fix misspelling in the YandexInstanceClass resource. (#1713)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d54ee9](https://github.com/deckhouse/deckhouse/commit/d54ee97647f7d5e8a8367ada99ada37a366ab608) in branch **main**:
```
[docs][linstor] Updates (#1677)

[docs][linstor] Documentation updates

Signed-off-by: Artem Kladov <artem.kladov@flant.com>

Co-authored-by: Andrei Kvapil <kvapss@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [381507](https://github.com/deckhouse/deckhouse/commit/3815075de24c079c2d6ac8ff1ce0827c1c0d0401) in branch **main**:
```
[docs] Fix broken links (#1715)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d62898](https://github.com/deckhouse/deckhouse/commit/d62898f4dda11f0371a7910985bca0e01f9d9ebd) in branch **main**:
```
[global] fix proxy doc (#1714)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [d05a94](https://github.com/deckhouse/deckhouse/commit/d05a94eb45b181e1356147c8c2261720179e35ab) in branch **main**:
```
[log-shipper] fix cluster-logging-config CRD (#1711)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c45197](https://github.com/deckhouse/deckhouse/commit/c4519796361c625019f67c99f34ed94aecd53a63) in branch **main**:
```
[cloud-provider-aws] fix terraform scheme (#1710)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [28b36b](https://github.com/deckhouse/deckhouse/commit/28b36bfbb0db2b87dba72ad6a189d1b1d3d3e3f5) in branch **main**:
```
[candi] update aws provider (#1716)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b03386](https://github.com/deckhouse/deckhouse/commit/b033868ac9eaff925e2cda940fa4f1f81b4cf714) in branch **main**:
```
[prometheus] rollback disk retention size to 80% (#1721)

Signed-off-by: Vitaliy Snurnitsin <vitaliy.snurnitsin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3f1f36](https://github.com/deckhouse/deckhouse/commit/3f1f3667119e80cad7d63567b4af686b37118082) in branch **main**:
```
[docs] Disaster recovery etcd doc (#1615)

[docs] etcd disaster recovery docs

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>

Co-authored-by: Konstantin Aksenov <konstantin.aksenov@flant.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
Co-authored-by: Kirill Kononovich <41591254+kirkonru@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [318471](https://github.com/deckhouse/deckhouse/commit/3184718ca72de5b7f215aa9f01587225cd8be91f) in branch **main**:
```
[docs][getting started] Add note about cloud provider quotas (#1723)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e544a6](https://github.com/deckhouse/deckhouse/commit/e544a67ec486ba905653bcaedb0a9f9f7e17d3fe) in branch **main**:
```
Re-generate changelog (#1706)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>

Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.32.12** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [3513e5](https://github.com/deckhouse/deckhouse/commit/3513e5fbb2e7429fb1ffb86a7c9253bb2853d888) in branch **main**:
```
[docs] Minor updates (#1725)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3b4f8e](https://github.com/deckhouse/deckhouse/commit/3b4f8e243f58f0ede5f3d40ae98a0fc36232aaa0) in branch **main**:
```
[upmeter] Add extensions probes (#1658)

Signed-off-by: Eugene Shevchenko <evgeny.shevchenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [eb0fc3](https://github.com/deckhouse/deckhouse/commit/eb0fc346befecda2aad2d3f34dacd821c68f08ce) in branch **main**:
```
[candi] fix build of registry packages (#1702)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [711370](https://github.com/deckhouse/deckhouse/commit/71137031b1692a99146b160497f124d4e32fb166) in branch **main**:
```
Add registry secret to ceph-csi module (#1733)

Signed-off-by: Eugene Shevchenko <evgeny.shevchenko@flant.com>
```
